tests/integration/sequential/test_par.py

```diff
@@ -385,3 +385,20 @@ def test_par_sequence_index_is_numerical():
     s1.fit(data)
     sample = s1.sample(2, 5)
     assert sample.columns.to_list() == data.columns.to_list()
+
+
+def test_par_error_on_context_columns():
+    metadata_dict = {
+        'columns': {
+            'A': {'sdtype': 'id'},
+            'B': {'sdtype': 'datetime', 'datetime_format': '%Y-%m-%d'},
+        },
+        'sequence_key': 'A',
```

Comment:
can we change `sequence_key` to something with more letters, if you do `set('A')` it will return what you are expecting but if you do `set('AB')` it will return `{'A', 'B'}`.

Reply:
Currently, sdv can only be a single column:
> The sequence key is a column that identify which row(s) belong to which sequences... At this time, SDV does not support composite keys.
- https://docs.sdv.dev/sdv/sequential-data/data-preparation/sequential-metadata-api#set_sequence_key
- Given this, I don't think it makes sense to make sequence_key multiple column names (which would imply composite keys)

Reply:
Yeah I was referencing my previous comment about the `set(str)` but since this is being automatically casted to iter by `ParSynthesizer` I think is good to go!